firefly_widgets/chart.py

```diff
@@ -0,0 +1,12 @@
+import ipywidgets as widgets
+from traitlets import Unicode
+
+
+@widgets.register
+class ChartViewer(widgets.DOMWidget):
+    """
+    Chart Viewer widget
+    """
+    _view_name = Unicode('ChartView').tag(sync=True)
+    _view_module = Unicode('jupyter-firefly').tag(sync=True)
+    tbl_group = Unicode().tag(sync=True)
```

Comment:
I have a question about tbl_group. Can it ever be set to anything but 'main'? I recall in the slate demo that all tables have to go into 'main' so I am wondering if this is related.

I tried setting tbl_group to 'tv' but then the widget would not re-display when I executed the cell with `cv`. Setting it back to 'main' allowed the widget to re-display properly again.